e2e/integration_tests/backends_test.ts

```diff
@@ -27,27 +28,28 @@ import {SMOKE} from './constants';
 /**
  *  This file tests backend switching scenario.
  */
-
+// TODO: Support backend switching between wasm and cpu.
+// https://github.com/tensorflow/tfjs/issues/7623
 describeWithFlags(
     `${SMOKE} backend switching`, {
-      predicate: testEnv => testEnv.backendName === 'webgl' &&
-          tfc.findBackend('webgl') !== null && tfc.findBackend('cpu') !== null
+      predicate: testEnv =>
+          testEnv.backendName !== 'cpu' && testEnv.backendName !== 'wasm'
```

Comment:
Do we need to use findBackend to ensure a specific backend is there?

Reply:
Currently, we may just extend the tests to WebGL/WebGPU <-> CPU

Reply:
I didn't see this is necessary for e2e tests.